app/config/config_prod.yml

```diff
@@ -28,5 +28,5 @@ monolog:
 
 doctrine:
     orm:
-        metadata_cache_driver: "%env(CACHE_DRIVER)%"
-        query_cache_driver:    "%env(CACHE_DRIVER)%"
+        metadata_cache_driver: "%cache.driver%"
```

Comment:
this way, the cache driver is not configurable using environment variables anymore, this is something we may want to keep, isn't it?

Reply:
I don't know if the actual word is "keep", because in my opinion this was never possible. As far as I understand, the value was **always** rewritten. We can add the feature, but my fix was sticking to the possible changes on 1.7.3.x:

https://github.com/PrestaShop/PrestaShop/blob/ff169adc50eb3519d2fdb83a28d8909bc87011e5/app/AppKernel.php#L63-L67

Reply:
ok ok :) go for setting value in container , dont forget to change "driver" by "array".

Reply:
but we need to change PR title :)

Reply:
done!